account/identity.go

```diff
@@ -142,22 +141,10 @@ func (m *GormIdentityRepository) Load(ctx context.Context, id uuid.UUID) (*Ident
 	return &native, errs.WithStack(err)
 }
 
-// Exists returns true|false where an object exists with an identifier
-func (m *GormIdentityRepository) Exists(ctx context.Context, id uuid.UUID) (bool, error) {
+// Exists returns true|false whether an identity exists with a specific identifier
+func (m *GormIdentityRepository) Exists(ctx context.Context, id string) (bool, error) {
 	defer goa.MeasureSince([]string{"goa", "db", "identity", "exists"}, time.Now())
```

Comment:
I understand your point but I prefer to keep all the calls in the same pkg. Besides this change will imply to pass a new param for the name of the entity.